addons/kommander/1.3/kommander.yaml

```
@@ -9,7 +9,7 @@ metadata:
     # This was originally added to support the PVC's needed for the Kubecost subcomponent.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.3.3-3"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.3.3-4"
```

Increment the addon patch version to account for the patch version change of the chart. Only increment release version when the only change is to the addon resource alone.

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.3.4-0"
```

---

I thought the addon revision version was supposed to match the kubeaddons-kommander release version (also `appversion`?) which is `1.3.3`. From what I see in KBA that's how it's done as well, unless we've all been doing it wrong this whole time 😅

---

we've been doing it wrong the whole time and we keep getting bit by it.

---

As discussed at standup, I didn't notice that this was a backport. Let's try to get this going on new PRs, not backports.

---

^ FYI @faiq for new autobump PRs across kba/kubeeaddons-kommander (although I suppose these won't be used in 2.x)

---

Wait so this is supposed to be chart version now?